integrated_channels/integrated_channel/models.py

```diff
@@ -671,6 +673,11 @@ class Meta:
         blank=True,
         null=True
     )
+    remote_errored_at = models.DateTimeField(
+        help_text='Date when the content transmission was failed in the remote API.',
+        blank=True,
+        null=True
+    )
```

Comment:
how are we ensuring that this field will always be populated? and what happens if we get a null. Plus aren't we require to backfill the value for this field for previously failed transmissions?

Reply:
I also thought first about backfilling but later I added this condition in the query
https://github.com/openedx/edx-enterprise/pull/2043/files/c6dedfbf9de37eccfdd2a391e77c303feb4dd531#diff-bd1856bd7ffab338eb216ef940de368327f49152f8673cc007eac15557220c78R207

Now basically we're selecting those records which errored 24hrs ago OR whose errored_at is `Null`

Reply:
Hmm, so this will work. But shouldn't we start populating it for future enrollments?

Reply:
It will populate for future enrollments - whenever an error occurs
https://github.com/openedx/edx-enterprise/pull/2043/files#diff-817b72970a2242a621a435d66ad70f35a223261ef3574f97a0ef5abcc1916530R254